nohup: ignoring input
[?1034hTClass::Init:0: RuntimeWarning: no dictionary for class ROOT::TIOFeatures is available
GluGluToContinToZZTo4e_13TeV_MCFM701_pythia8 nevents 977600.0 sumw 977600.0
GluGluToContinToZZTo4mu_13TeV_MCFM701_pythia8 nevents 847600.0 sumw 847600.0
GluGluToContinToZZTo2e2mu_13TeV_MCFM701_pythia8 nevents 494000.0 sumw 494000.0
ZZTo4L_TuneCP5_13TeV_powheg_pythia8 nevents 6689900.0 sumw 6622242.0
data_13TeV_2017_duplicated_removed nevents 0.0 sumw 0.0
results for observable==== mass4l and channel===== 4mu  +++++++++++++++++++++++++++++++++++++++++ 

==========================================
4mu mass4l 105 - 140 GeV
EventCat== -1
GluGluToContinToZZTo4e_13TeV_MCFM701_pythia8
ggZZ weight is=================== (genWeight*crossSection*dataMCWeight*pileupWeight*58800.0/977600.0)*(k_ggZZ)
GluGluToContinToZZTo4e_13TeV_MCFM701_pythia8 ======== 0.0
GluGluToContinToZZTo4mu_13TeV_MCFM701_pythia8
ggZZ weight is=================== (genWeight*crossSection*dataMCWeight*pileupWeight*58800.0/847600.0)*(k_ggZZ)
GluGluToContinToZZTo4mu_13TeV_MCFM701_pythia8 ======== 3.85805764479
GluGluToContinToZZTo2e2mu_13TeV_MCFM701_pythia8
ggZZ weight is=================== (genWeight*crossSection*dataMCWeight*pileupWeight*58800.0/494000.0)*(k_ggZZ)
GluGluToContinToZZTo2e2mu_13TeV_MCFM701_pythia8 ======== 0.0
ZZTo4L_TuneCP5_13TeV_powheg_pythia8
ZZTo4L_TuneCP5_13TeV_powheg_pythia8 ======== 38.0329665095
data_13TeV_2017_duplicated_removed
data_13TeV_2017_duplicated_removed ======== 85.0

ZZ Bkg:   41.8910241543
ggZZ Bkg:   3.85805764479
qqZZ Bkg:   38.0329665095
Red Bkg:   12.8790159822
Data  :   85.0

Error in <TPad::TPad>: You must create a TCanvas before creating a TPad
Info in <TCanvas::MakeDefCanvas>:  created default TCanvas with name c1
TPad::ResizePad:0: RuntimeWarning:  width changed from 0 to 10

TPad::ResizePad:0: RuntimeWarning:  height changed from 0 to 10

Traceback (most recent call last):
  File "HZZPlots_80X_combine.py", line 423, in <module>
    plot_m4l(chan, observable, '125', 40, bins[0],bins[1], 105.0, 140.0, 'm_{4l}', 'GeV', True, False, False,"-1")    
  File "HZZPlots_80X_combine.py", line 385, in plot_m4l
    c1.SaveAs('Histo_' + save + '.pdf')
AttributeError: 'PyROOT_NoneType' object has no attribute 'SaveAs'
